src/panels/GeoLocationPanel/Search/util.ts

```diff
@@ -0,0 +1,59 @@
+import { computeDistanceBetween, LatLng } from 'spherical-geometry-js';
+
+import { ArcGISJSON, Extent, MatchedLocation } from './types';
+
+export const ET_URL = 'http://localhost:3000/1/search/';
```

Comment:
```suggestion
export const ET_URL = 'https://geocode.openspaceproject.com/1/search/';
```

Reply:
Realised from other files that is it Extra Terrestrial :P  But still, please write that out in this variable name